src/main/resources/locale/messages.properties

```diff
@@ -1,3 +1,4 @@
 error.name.notnull = Name can not be null.
 error.content.notnull = Content can not be null.
-error.image.notnull = Image can not be null.
\ No newline at end of file
+error.image.notnull = Image can not be null.
+notfound = with that id not found
```

Comment:
te falto un "was" antes de "not", y deberias separarlo como los anteriores, con un punto -> `not.found = wit that id was not found`